Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM python:3.8-slim AS conpot-builder
```

Comment:
```suggestion
FROM python:3.8 AS conpot-builder
```
The regular version comes with gcc, saving the install step below. However, the initial download is much bigger (900 MB vs. 120 MB), and the run container would still be `-slim`. So probably not worth the switch.